fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBStoreTimer.java

```diff
@@ -570,6 +570,10 @@ public String logKey() {
         ONLINE_INDEX_BUILDER_RECORDS_SCANNED("number of records scanned by online index build", false),
         /** The number of records indexed by {@link OnlineIndexer}. */
         ONLINE_INDEX_BUILDER_RECORDS_INDEXED("number of records indexed by online index build", false),
+        /** The number of {@link OnlineIndexer} records scanned in a single iteration/transaction. */
+        ONLINE_INDEX_BUILDER_RANGE_BY_COUNT("number of indexer iterations terminated by scan limit", false),
+        /** The number of {@link OnlineIndexer} bytes to write in a single iteration/transaction. */
```

Comment:
Shall I use: 
```
previous comment for number of records scanned (as it was): 
 /** The number of records indexed by {@link OnlineIndexer}. */
and the new size limitation (as you suggested in earlier comment): 
 /** The number of {@link OnlineIndexer} range scans terminated after hitting the scan limit. */
```
? 
If not, would you suggest another comment(s)?

Reply:
Yeah, I like `The number of {@link OnlineIndexer} range scans terminated after hitting the scan limit.` (and similar for `ONLINE_INDEX_BUILDER_RANGE_BY_COUNT`.

Although, actually, the names here should probably be `RANGES_BY_COUNT` and `RANGES_BY_SIZE` (plural instead of singular).